service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -185,27 +188,25 @@ public void deletingAnExistingPolicy() throws Exception {
   public RequestResponsePact getPaoWithAnExistingPolicy(PactDslWithProvider builder) {
     var policyResponseShape =
         new PactDslJsonBody()
-            .valueFromProviderState("objectId", existingPolicyProviderStateValue, existingPolicyId)
+            .stringValue("objectId", existingPolicyId.toString())
```

Comment:
This works with:
`.valueFromProviderState("objectId", existingPolicyProviderStateValue, existingPolicyId.toString())`
instead of
`.valueFromProviderState("objectId", existingPolicyProviderStateValue, existingPolicyId)`

Apparently the uuid isn't getting converted to a string automatically.